tests/test_driver.py

```diff
@@ -133,3 +136,41 @@ def test_status_not_found(self, driver):
         txid = 'dummy_id'
         with raises(NotFoundError):
             driver.transactions.status(txid)
+
+    def test_prepare(self, driver, alice_pubkey):
+        transaction = driver.transactions.prepare(owners_before=[alice_pubkey])
```

Comment:
Maybe this could be a fixture that can then also be used by `test_fulfill` rather than the hardcoded `unsigned_transaction`?